src/app/pages/credentials/new-users/all-users/user-details/user-password-card/user-password-card.component.html

```diff
@@ -0,0 +1,37 @@
+<mat-card class="card">
+  <mat-card-header>
+    <h3 mat-card-title>
+      {{ 'Password' | translate }}
+    </h3>
+  </mat-card-header>
+
+  <mat-card-content>
+    <p class="row">
+      <span class="label">{{ 'Change Required' | translate }}:</span>
+      <span class="value">{{ user()?.password_change_required | yesNo }}</span>
+    </p>
+
+    <p class="row">
+      <span class="label">{{ 'Age' | translate }}:</span>
+      <span class="value">{{ !user().password_age && user().password_age !== 0
+        ? '–'
+        : '{n, plural, =0 {Less than a day} one {# day} other {# days} }' | translate: {
+          n: user().password_age,
+        }
+        }}</span>
+    </p>
+
+    <p class="row">
+      <span class="label">{{ 'History' | translate }}:</span>
+      <span class="value">{{ '{n, plural, =0 {No History} one {# entry} other {# entries}}' | translate :
+        { n: user().password_history.length }
```

Comment:
I tried it on m50 machine, there are errors, would be good to prevent them.
`{ n: user().password_history?.length }`

<img width="650" alt="Screenshot 2025-05-06 at 09 40 43" src="https://github.com/user-attachments/assets/197acf06-3535-40f2-8fc2-09db6891ef85" />